src/main/java/hudson/plugins/pmd/PmdPublisher.java

```diff
@@ -25,14 +24,20 @@
  * @author Ulli Hafner
  */
 public class PmdPublisher extends HealthAwarePublisher {
+
     private static final long serialVersionUID = 6711252664481150129L;
```

Comment:
Nope. XStream doesn't care about serialVersionUID. Remoting uses java serialization and thus if you end up with a different JVM on the other side that may calculate a different default value and cause a specific slave to fail to be able to capture PMD results for example